commit ee21397ca3b4faca671d42d4a252ad80d45897ed (refs/changes/47/847/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2016-11-16T13:21:27+00:00 (2 years, 11 months ago)
    
    PUBAPI-1345: Server hangs on close when started from some tests

diff --git a/lib/app.js b/lib/app.js
index 319c2c4..86f59fa 100644
--- a/lib/app.js
+++ b/lib/app.js
@@ -691,6 +691,8 @@ module.exports = {
                 server.on('close', function () {
                     clearInterval(globalAgentInterval);
 
+                    kangServer.close();
+
                     clients.vmapi.close();
                     clients.napi.close();
                     clients.fwapi.close();
